gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -1292,6 +1326,26 @@ MM_ParallelGlobalGC::fixHeapForWalk(MM_EnvironmentBase *env, UDATA walkFlags, ui
 	return fixedObjectCount;
 }
 
+void
+MM_ParallelGlobalGC::clearHeap(MM_EnvironmentBase *env, MM_HeapWalkerObjectFunc walkFunction)
```

Comment:
method description is missing (Doxygen format): https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-function-parameters-clearly